.github/workflows/my_CI_pipelines.yml

```diff
@@ -28,8 +28,5 @@ jobs:
       - name: Build Docker Image for Tests
```

Comment:
I can't comment on the relevant line because it was merged in already, but we don't need the `services` section:
```yml
    services:
      docker:
        image: docker:latest
        ports:
          - 2375:2375
```
This sets up a container that is designed for _other_ docker containers to be able to talk to easily (e.g. [the example here](https://docs.github.com/en/actions/using-containerized-services/about-service-containers) sets up a redis DB so the other containers run as part of the pipeline can rely on that) but our tests don't rely on any other containers